File: code-recipes/java/src/test/java/aplus/UploadImageForResourceTest.java
package aplus;

import util.RecipeTest;

import java.util.List;

public class UploadImageForResourceTest extends RecipeTest {

    protected UploadImageForResourceTest() {
        super(
                new UploadImageForResourceRecipe(),
                List.of(
                    "aplus-createUploadDestinationResponse"
                )
        );
    }
}

File: code-recipes/php/test/aplus/UploadImageForResourceTest.php
<?php

namespace Test\aplus;

use Src\aplus\UploadImageForResourceRecipe;
use Test\util\RecipeTestCase;

class UploadImageForResourceTest extends RecipeTestCase
{
    protected function setUp(): void
    {
        $this->recipe = new UploadImageForResourceRecipe();
        $this->responses = [
            "aplus-createUploadDestinationResponse"
        ];
    }
}
